### ISSUE TITLE
v-model textarea not working

### ISSUE THREAD

**Issue by ZavilenskyIlya** [fritx/vue-at#96]
This is not a working code for version 2.5.

```
<template>
  <at-ta :members="members"  v-text="text">
    <textarea class="editor"></textarea>
  </at-ta>
</template>

<script>
import AtTa from 'vue-at/dist/vue-at-textarea'

export default {
  components: { AtTa },
  data () {
    return {
      text: 'Hello World! @Roxie Miles .\n@grace.carroll @小浩 lol',
      members: ['Roxie Miles', 'grace.carroll', '小浩']
    }
  }
}
</script>
```

**Comment by JoshuaeKaiser** [comment]
any fix for this? makes the plug in pretty much unusable otherwise...

**Comment by fritx** [comment]
@ZavilenskyIlya @JoshuaeKaiser sorry for the late response, but v-model should work,
for textarea, contenteditable and even vuetify..
See the documentation: https://fritx.github.io/vue-at/#/en/vmodel

You can paste your code here we would help as well.

**Comment by JoshuaeKaiser** [comment]
hi @fritx thanks for the reply. Its a great component but im having a error around vue model that i think may be tied to the version number of VUE perhaps? as the code is identical to the demo but the result is different. When an option from the mentions is selected, the option gets added to the textarea like normal, however the vue model doesnt get updated until the user either types something additional or preses the space bar/return key. 

Its also worth noting that the only way i could even get it to update the vue model at all was by putting vue model on the text area element and not the <at-ta>, like this 

<at-ta :ats="atTrigger"  :members="someArray">
    <textarea  v-model="someVar"></textarea>
</at-ta>

___________

below doesnt work at all

<at-ta :ats="atTrigger"  :members="someArray"  v-model="someVar">
    <textarea></textarea>
</at-ta>

any idea how i can fix the issue where the v model doesnt get auto updated on select of one of the mentions ?

many thanks

**Comment by fritx** [comment]
@JoshuaeKaiser thanks for use this library.
Usage here is recommended to apply v-model onto the `<at-ta>` 'coz it manipulate the dom directly by inserting something, and it's likely to not have the v-model updated if it was applied on a `<textarea>` instead

Which version of Vue are you using?

**Comment by JoshuaeKaiser** [comment]
@fritx the version of Vue im using is **3.5.3**. I have tried both, if i inser it on the <at-ta> it does not work at all. If i insert it on the <textarea> it works, but will only update the V-model after I have either presed the space key or typed something else in. This is fine but not very user friendly as if you insert something at the end of a block of text and then say go to save that text by grabing the associated key in data(){} the selected value will be missing unles you remebered to press the space key or types something else.

**Comment by fritx** [comment]
You are right, but it is supposed that v-model is working well with `<at>` and `<at-ta>`, let's figure out what was going wrong.

@JoshuaeKaiser you mean Vue 3.5.3 or Vue-Cli 3.5.3?
https://github.com/vuejs/vue/releases
https://github.com/vuejs/vue-cli/releases

Looks like that there is no 3.x release in Vue itself, are you sure about that?

**Comment by JoshuaeKaiser** [comment]
@fritx apologies you are right thats my CLI version.

"textarea-caret": "^3.1.0",
"vue": "^2.6.6",
"vue-at": "^2.3.2",

these are the versions of all relative depedencies. 

this is what i have:

`<at-ta  :ats="atTrigger" :members="slotList">
                  <el-input v-model="responseForm.text" type="textarea"></el-input>
                </at-ta>`

^ this works to a degree. I can select and it puts the value in the text area, however it will not update the vue model until i type or press the space bar.

I have also tried this:

`<at-ta v-model="responseForm.text" :ats="atTrigger" :members="slotList">
                  <el-input  type="textarea"></el-input>
                </at-ta>`

^ this does not work at all. 

**NOTE: i have also tried the same in bot cases with regular input and not just my own custom input compotents <textarea></textarea> --> same exact issue in bot cases

**Comment by fritx** [comment]
@JoshuaeKaiser ooh are you using `<el-input>` from element-ui?
Have never tried that yet, will look into it then.

**Comment by JoshuaeKaiser** [comment]
@fritx yes i am. but i have also tested it with a regular old <textarea></textarea> and I have the same issue.

**Comment by fritx** [comment]
![image](https://user-images.githubusercontent.com/6647633/58363248-dba91000-7ed3-11e9-9523-8558c5ba2255.png)

https://github.com/fritx/vue-at#vuetify-v-textarea

**Comment by Abdulsametileri** [comment]
Not working at all. :/

**Comment by fritx** [comment]
It's recommended to use like

```vue
<el-input type="textarea" v-model="input">
```

Or if you need one-line input-box, try
```vue
<el-input type="textarea" :rows="1" resize="none" v-model="input">
```

(Adding resize=none can disable user-resizing as default behavior)

For more detail, see https://github.com/fritx/vue-at/issues/112#issuecomment-616152442

**Comment by bigperson** [comment]
@JoshuaeKaiser I have the same problem. Have you found a solution?
